--- ../mediawiki-1.13.1/includes/SearchEngine.php	2008-07-29 01:47:57.000000000 +0200
+++ /home/www/mediawiki-1.13.1/includes/SearchEngine.php	2008-09-28 01:14:33.000000000 +0200
@@ -71,6 +71,8 @@
 				return $title;
 			}
 
+			if( !wfRunHooks( 'SearchEngineAfterNoDirectMatch', array( $term, &$title ) ) ) return $title;
+
 			# Now try all lower case (i.e. first letter capitalized)
 			#
 			$title = Title::newFromText( $wgContLang->lc( $term ) );
